CMakeLists.txt

```diff
@@ -278,6 +279,12 @@ if(HARDENING)
   target_link_libraries(core INTERFACE hardening)
 endif()
 
+if(REDUCE_EXPORTS)
```

Comment:
Note that this should probably become on by default in the future (once libbitcoinkernel is ok with it).

Historically it was only off by default because of an ancient broken boost impl that didn't properly export its exceptions.